samples/demo/src/db/repository.ts

```diff
@@ -18,51 +19,65 @@ export interface Repository {
+        console.log("Using MongoDB: found connection string in environment variable CONNECTION_MONGODB_CONNECTIONSTRING");
+        return new MongoFactory(process.env.CONNECTION_MONGODB_CONNECTIONSTRING);
     }
 
     if (process.env.CONNECTION_REDIS_URL) {
-      console.log("Using Redis: found url in environment variable CONNECTION_REDIS_URL");
-      return new RedisFactory(process.env.CONNECTION_REDIS_URL);
+        console.log("Using Redis: found url in environment variable CONNECTION_REDIS_URL");
+        return new RedisFactory(process.env.CONNECTION_REDIS_URL);
     }
 
     if (process.env.CONNECTION_REDIS_HOST) {
-      console.log("Using Redis: found hostname in environment variable CONNECTION_REDIS_HOST");
-      const connection = { 
-        host: process.env.CONNECTION_REDIS_HOST!, 
-        port: process.env.CONNECTION_REDIS_PORT!,
-        username: process.env.CONNECTION_REDIS_USERNAME || '',
-        password: process.env.CONNECTION_REDIS_PASSWORD || '',
-      }
-  
-      let scheme = "redis"
-      if (connection.port === "6380") {
-        scheme = "rediss"
-      }
-  
-      let usernamePass = "";
-      if (connection.username !== "" || connection.password !== "") {
-        usernamePass = `${connection.username}:${connection.password}@`
-      }
-  
-      const url = `${scheme}://${usernamePass}${connection.host}:${connection.port}`
-      return new RedisFactory(url);
+        console.log("Using Redis: found hostname in environment variable CONNECTION_REDIS_HOST");
+        const connection = {
+            host: process.env.CONNECTION_REDIS_HOST!,
+            port: process.env.CONNECTION_REDIS_PORT!,
+            username: process.env.CONNECTION_REDIS_USERNAME || '',
+            password: process.env.CONNECTION_REDIS_PASSWORD || '',
+        }
+
+        let scheme = "redis"
+        if (connection.port === "6380") {
+            scheme = "rediss"
+        }
+
+        let usernamePass = "";
+        if (connection.username !== "" || connection.password !== "") {
+            usernamePass = `${connection.username}:${connection.password}@`
+        }
+
+        const url = `${scheme}://${usernamePass}${connection.host}:${connection.port}`
+        return new RedisFactory(url);
     }
-  
+
+    if (process.env.CONNECTION_POSTGRES_HOST) {
+        console.log("Using PostgreSQL: found hostname in environment variable CONNECTION_POSTGRES_HOST");
+        const connection = {
+            host: process.env.CONNECTION_POSTGRES_HOST!,
+            port: process.env.CONNECTION_POSTGRES_PORT!,
+            username: process.env.CONNECTION_POSTGRES_USERNAME || '',
+            password: process.env.CONNECTION_POSTGRES_PASSWORD || '',
+            database: process.env.CONNECTION_POSTGRES_DATABASE || '',
+        }
+        const url = `postgresql://${connection.username}:${connection.password}@${connection.host}:${connection.port}/${connection.database}`
+        console.log(`Postgres URL: ${url}`);
```

Comment:
Was this log statement just added for local debugging? (should remove this since it includes password)